lib/exposes.js

```diff
@@ -82,15 +82,15 @@ class Lock extends Base {
         this.features = [];
     }
 
-    withState(property, valueOn, valueOff, description, access=a.ALL) {
+    withState(property, valueOn, valueOff, description, access=a.ALL, label='state') {
         assert(!this.endpoint, 'Cannot add feature after adding endpoint');
-        this.features.push(new Binary('state', access, valueOn, valueOff).withProperty(property).withDescription(description));
+        this.features.push(new Binary(label, access, valueOn, valueOff).withProperty(property).withDescription(description));
```

Comment:
I don't think the extra argument is necessary. Looks more clean to just pass `property` to the first argument of `new Binary()`.

Reply:
That was what I first had, I'll change it back.